packages/stack-deploy-iam/bin/app.ts

```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env node
 import * as cdk from '@aws-cdk/core';
 import * as ssm from '@aws-cdk/aws-ssm';
-import { 
-   PolicyStatement, 
-   Effect,
-   Group,
-   User
+import { readFileSync } from "fs";
+import {
+    PolicyStatement,
+    Effect,
+    Group,
+    User
 } from '@aws-cdk/aws-iam';
 
 const STACK_NAME = process.env.STACK_NAME
+const CUSTOM_POLICY_PATHS = process.env.CUSTOM_POLICY?.split(',');
```

Comment:
Instead of specifying multiple files and splitting like this, I think it would be better to just have one file that contains an array of policies. 

Then you can just loop over those and add to the group

Reply:
:+1: Although not a bad idea to allow multiple files as well right? Might as well provide the option for multiple policies in multiple files.

Reply:
Great I will look into that :+1: